src/rules/media-feature-name-case/__tests__/index.js

```diff
@@ -0,0 +1,330 @@
+    code: "@media ('MIN-' + $WIDTH: $value) { }",
+  }, {
+    code: "@media ($value + 'width': $value) { }",
+  }, {
+    code: "@media ($VALUE + 'WIDTH': $value) { }",
+  }, {
+    code: "@media (#{$width}: $value) { }",
+  }, {
+    code: "@media (#{$WIDTH}: $value) { }",
+  }, {
+    code: "@media #{$feature-name} { }",
+  } ],
+
+  reject: [ {
+    code: "@media (MIN-WIDTH: $var) { }",
+    message: messages.expected("MIN-WIDTH", "min-width"),
+    line: 1,
+    column: 9,
+  }, {
+    code: "@media (MIN-WIDTH: $var + 10px) { }",
+    message: messages.expected("MIN-WIDTH", "min-width"),
+    line: 1,
+    column: 9,
+  }, {
+    code: "@media (MIN-WIDTH: ($var + 10px)) { }",
+    message: messages.expected("MIN-WIDTH", "min-width"),
+    line: 1,
+    column: 9,
+  } ],
+})
+
+testRule(rule, {
+  ruleName,
+  config: ["upper"],
+
+  accept: [ {
+    code: "@media not all and (MONOCHROME) { }",
+  }, {
+    code: "@media (MIN-WIDTH: 700px) { }",
+  }, {
+    code: "@media (MIN-WIDTH: 700PX) { }",
+  }, {
+    code: "@media (MIN-WIDTH: 700px) and (ORIENTATION: landscape) { }",
+  }, {
+    code: "@media (MIN-WIDTH: 700px), print and (ORIENTATION: landscape) { }",
+  }, {
+    code: "@media (MIN-WIDTH: 700px), PRINT and (ORIENTATION: landscape) { }",
+  }, {
+    code: "@media (-WEBKIT-MIN-DEVICE-PIXEL-RATION: 2) { }",
+  }, {
+    code: "@not-media (min-width: 700px) { }",
+    description: "ignore non media at-rule",
+  }, {
+    code: "@media (--viewport-medium) { }",
+    description: "ignore css variables",
+  }, {
+    code: "@media (--VIEWPORT-MEDIUM) { }",
+    description: "ignore css variables",
+  } ],
+
+  reject: [ {
+    code: "@media not all and (monochrome) { }",
+    message: messages.expected("monochrome", "MONOCHROME"),
+    line: 1,
+    column: 21,
+  }, {
+    code: "@media not all and (mOnOcHrOmE) { }",
+    message: messages.expected("mOnOcHrOmE", "MONOCHROME"),
+    line: 1,
+    column: 21,
+  }, {
+    code: "@media (min-width: 700px) { }",
+    message: messages.expected("min-width", "MIN-WIDTH"),
+    line: 1,
+    column: 9,
+  }, {
+    code: "@media (mIn-WiDtH: 700px) { }",
+    message: messages.expected("mIn-WiDtH", "MIN-WIDTH"),
+    line: 1,
+    column: 9,
+  }, {
+    code: "@media (min-width: 700px) and (ORIENTATION: landscape) { }",
+    message: messages.expected("min-width", "MIN-WIDTH"),
+    line: 1,
+    column: 9,
+  }, {
+    code: "@media (MIN-WIDTH: 700px) and (orientation: landscape) { }",
+    message: messages.expected("orientation", "ORIENTATION"),
+    line: 1,
+    column: 32,
+  }, {
+    code: "@media (-webkit-min-device-pixel-ration: 2) { }",
+    message: messages.expected("-webkit-min-device-pixel-ration", "-WEBKIT-MIN-DEVICE-PIXEL-RATION"),
+    line: 1,
+    column: 9,
+  } ],
+})
+
+testRule(rule, {
+  ruleName,
+  config: ["upper"],
+  syntax: "less",
+
+  accept: [ {
+    code: "@media (MIN-WIDTH: @tablet) { }",
+  }, {
+    code: "@media (MIN-WIDTH: (@value + 10px)) { }",
```

Comment:
Why are there these tests that are equivalent to other tests but with extra parentheses? What do they add?